NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="Aspnet Release Old" value="https://myget.org/f/aspnetrelease" />
```

Comment:
nuget.org has the corrupted bit (the specified version is not available in nuget.org), which is why I have added aspnetrelease feed as well so that it brings the specific version.